pkgs/development/ocaml-modules/ocb-stubblr/default.nix

```diff
@@ -13,9 +13,9 @@ stdenv.mkDerivation rec {
   patches = [ ./pkg-config.patch ];
 
   nativeBuildInputs = [ ocaml findlib ocamlbuild topkg ];
-  buildInputs = [ topkg ocamlbuild ];
+  buildInputs = [ topkg ];
 
-  propagatedBuildInputs = [ astring ];
+  propagatedBuildInputs = [ astring ocamlbuild ];
```

Comment:
Why treating differently topkg and ocamlbuild?

Reply:
You are correct, fixed